### ISSUE TITLE
Custom Updater is not working 

### ISSUE THREAD

**Issue by humphrep** [jrlucier/eero_tracker#5]
I tried to add the custom updater component and it isn't showing up in my card.  I have other custom components that do show up, so I believe I have my side set up correctly.

This is the url I am using:

https://raw.githubusercontent.com/jrlucier/eero_tracker/master/custom_updater.json

Here are the current contents of that file:
{
    "device_tracker.eero_tracker": {
      "updated_at": "2019-04-21",
      "version": "1.0.4",
      "local_location": "/custom_components/eero_tracker/device_tracker.py",
      "remote_location": "https://raw.githubusercontent.com/jrlucier/eero_tracker/master/custom_components/eero_tracker/device_tracker.py",
      "visit_repo": "https://github.com/jrlucier/eero_tracker",
      "changelog": "https://github.com/jrlucier/eero_tracker/releases"
    }
}

**Comment by humphrep** [comment]
I forked off a branch and made two changes and now it is working when pointing to my branch.

1st - I changed the custom_updater.json file and changed "device_tracker.eero_tracker" to just "eero_tracker".

2nd - I added   VERSION = '1.0.4'   to the device_tracker.py file just below this line:
REQUIREMENTS = ['requests==2.13.0']

Then after restarting HA, I now see eero_tracker in my card, and it shows my current version is 1.0.4 and the available version is 1.0.4.
